src/client/client.cpp

```diff
@@ -216,9 +216,13 @@ void Client::processServerInput(boost::asio::io_context& context) {
 }
 
 void Client::draw() {
+    glm::vec3 test(1.0f);
+
     for (int i = 0; i < this->gameState.objects.size(); i++) {
         std::shared_ptr<SharedObject> sharedObject = this->gameState.objects.at(i);
 
+        std::cout << "shared object " << i << ": position: " << glm::to_string(sharedObject->physics.position) << std::endl;
+
         if (sharedObject == nullptr)
             continue;
```

Comment:
nitpick: if we want to keep this position printout, maybe move to after checking if `sharedObject == nullptr` just in case